packages/server/src/server.js

```diff
@@ -214,9 +215,9 @@ export default class PostRPCServer {
 			jsonrpc,
 			id: request.id,
 			error: {
-				code: invalidParamsCode,
-				message: invalidParamsMessage,
-				data: invalidParamsData
+				code: invalidArgsCode,
+				message: invalidArgsMessage,
+				data: invalidArgsData
```

Comment:
method name - `invalidArgsResponse` ? instead of invalidParamsResponse

Reply:
right, because an arg is what is passed. a param is what is defined.

Reply:
I renamed all of them accordingly.

Reply:
right, you missed the method name

Reply:
sorry, it doesnt let me comment on lines that werent changed

Reply:
fixed. thanks.